docs/external-module.format.md

```diff
@@ -0,0 +1,24 @@
+# External Module Format
+Typings are written in non-ambient external module format. Most of the time we refer it simply as external module format.
```

Comment:
Only thoughts, but here's how I might phrase it (forgive me as I keep doing this to you):

> Typings are written in the TypeScript external module format, by default. Also referred to as "non-ambient".